overlays/ps-nlp-tensorflow-gpu/Pipfile

```diff
@@ -8,6 +8,7 @@ nltk = "*"
 gensim = "*"
 spacy = "*"
 keras = "*"
+jupyter-tensorboard = "*"
```

Comment:
why does this need to be a direct dependency?

Reply:
isn't it a transitive dependency? It should be part of an advise this information?

Reply:
sorry, I m not aware if it is a transitive dependency of any package, can you please share the package :grinning: 

>It should be part of an advise this information?
you mean by thoth advise.

Reply:
I see you are adding `jupyter-tensorboard`, is there an error in the build process if this is missing? or is there any other reason for having that? If that is the case, should that information be part of a reccomendation, in other words, should we have a new justification?

Reply:
this is not based on error.
this is a plugin for tensorboard to show up in the jupyter.